doc_src/cmds/while.rst

```diff
@@ -6,7 +6,7 @@ while - perform a set of commands multiple times
 Synopsis
 --------
 
-**while** *CONDITION*; *COMMANDS...*; **end**
+**while** *CONDITION*; *COMMANDS*; **end**
```

Comment:
`if` has `*COMMANDS_TRUE* ...`.

Same point of showing this as multiline applies. Also should this be using `|` at the beginning of the line?

Reply:
Using it adds the "multiline" class in the HTML output, which might help with CSS.